packages/core/tocco-util/src/tqlBuilder/tqlBuilder.js

```diff
@@ -83,7 +83,7 @@ const typeHandlers = type => {
     case 'boolean':
       return (path, value) => (isDefined(value) ? `${path} == ${value}` : null)
     case 'marking':
-      return (path, value) => (value === false ? null : `exists(${path})`)
+      return (path, value) => (value === false ? `not exists(${path})` : `exists(${path})`)
```

Comment:
Nur zur Sicherheit, wenn in der Select-Box gar nichts angewählt kommt es hier gar nicht rein, korrekt?

Reply:
Richtig